runtime/centrifuge/src/lib.rs

```diff
@@ -767,8 +767,14 @@ impl pallet_fees::Config for Runtime {
 	type WeightInfo = weights::pallet_fees::SubstrateWeight<Self>;
 }
 
+parameter_types! {
+	pub const PreCommitDeposit: Balance = 100 * CFG;
```

Comment:
I this value final? This is _a lot_ 🤔

Reply:
I have used the same used in the keystore pallet, but it is a value that definitely we need to discuss. I add it to the question list 😛

Reply:
Finally we've chosen 1 CFG :)